packages/app/components/drop/drop.tsx

```diff
@@ -10,7 +10,7 @@ const STORE_KEY = "showExplanation";
 
 export const Drop = () => {
   const [showExplanation, setShowExplanation] = useState(
-    () => store.getBoolean(STORE_KEY) ?? true
+    () => store.getBoolean(STORE_KEY + "v2") ?? true
```

Comment:
Is the anon function intentional here for the default state value? I don't see any reason for this

Reply:
yeah, it's nit but better for performance. It'll be only called when this component renders for the first time whereas if we do `store.getBoolean(STORE_KEY + "v2")`, it'll be called on every render although react will use the first render's value

Reply:
actually it missed to set `STORE_KEY + "v2"` key, so it alway is return `undefined`.  

https://user-images.githubusercontent.com/37520667/218055622-746876fc-5dd8-4dc9-90b1-ab3acdad4853.mp4

Reply:
fixed it! 
![CleanShot 2023-02-10 at 17 32 26](https://user-images.githubusercontent.com/37520667/218056162-69cacdf9-2b33-471f-95cc-e2cc65775ef3.png)

Reply:
damn. thanks 🐐

Reply:
> yeah, it's nit but better for performance. It'll be only called when this component renders for the first time whereas if we do store.getBoolean(STORE_KEY + "v2"), it'll be called on every render although react will use the first render's value


TIL, thanks!